OpenRA.Mods.Common/Traits/Harvester.cs

```diff
@@ -366,6 +370,22 @@ int ISpeedModifier.GetSpeedModifier()
 			return 100 - (100 - Info.FullyLoadedSpeed) * contents.Values.Sum() / Info.Capacity;
 		}
 
+		protected override void TraitEnabled(Actor self)
+		{
+			if (Info.SearchOnCreation)
+				self.QueueActivity(new FindAndDeliverResources(self));
```

Comment:
Are we sure this doesn't cause the activity to be queued twice when the trait starts enabled? (Once in `Created` and once here.)

Reply:
I'm not. I'm not sure about this at the first place tbh. But if we want it to be triggered when enabled, one in the Created can go. Tho it was crashing when you leave the game when this was in a EndFrameTask and seems to be needed in Created as the comment says.

Reply:
Yeah, i checked. It crashes with EndFrameTask and comment still applies without it. So not sure how to go with this.

Reply:
I would simply remove this from `TraitEnabled`.